packages/orbit-components/src/Wizard/WizardStep.tsx

```diff
@@ -180,9 +68,13 @@ const WizardStep = ({ dataTest, title, onClick, isCompleted }: WizardStepProps)
             title
           )}
         </ButtonLink>
-        {isActive && <StyledActiveMarker />}
-        {status !== "disabled" && <StyledBorder />}
-      </StyledContainer>
+        {isActive && (
+          <div className="w-xxxs bg-product-normal rounded-e-normal absolute inset-y-px ltr:left-0 rtl:right-0" />
+        )}
+        {status !== "disabled" && (
+          <div className="border-cloud-normal ms-xxl absolute inset-x-0 bottom-0 border-t border-solid" />
```

Comment:
Exactly. I wanted to use logical properties instead of `left`. Even though we could’ve kept `left` since it doesn’t matter for RTL in this case I think it’s better to avoid it because anyone who’ll go through the code will be wondering if we shouldn’t handle `rtl:` there as well.